app/serializers/project_serializer.rb

```diff
@@ -126,6 +126,10 @@ def avatar_src
     end
   end
 
+  def active_workflows
+    @model.workflows.where(active: true)
```

Comment:
This may result in a query and an N+1 in the serializer page method. You should just be able to use the project#active_workflows relation [here](https://github.com/zooniverse/Panoptes/blob/91ec95d77b682dd4e0715d9bdff64b225e4fb949/app/models/project.rb#L21) instead of manually running the query. If you do that then you should also preload that relation in the serializer page [here](https://github.com/zooniverse/Panoptes/blob/91ec95d77b682dd4e0715d9bdff64b225e4fb949/app/serializers/project_serializer.rb#L54)

Maybe even migrate that to use the https://github.com/zooniverse/Panoptes/blob/91ec95d77b682dd4e0715d9bdff64b225e4fb949/lib/serialization/panoptes_restpack.rb as well. You may still need to keep the page class method override to get the cards preload sorted too.